partitioned-table.md

```diff
@@ -1049,6 +1235,6 @@ select * from t;
 5 rows in set (0.00 sec)
 ```
 
-The `tidb_enable_table_partition` environment variable controls whether to enable the partitioned table feature. If this variable is set to `off`, the partition information will be ignored when a table is created, and this table will be created as a normal table.
+The `tidb_enable_table_partition` environment variable controls whether to enable the partitioned table feature. If this variable is set to `OFF`, the partition information will be ignored when a table is created, and this table will be created as a normal table.
```

Comment:
According to the original PR, the variable name here is changed to `tidb_enable_list_partition`.
```suggestion
The `tidb_enable_list_partition` environment variable controls whether to enable the partitioned table feature. If this variable is set to `OFF`, the partition information will be ignored when a table is created, and this table will be created as a normal table.
```